test/unit/org/apache/cassandra/index/sai/cql/VectorHybridSearchTest.java

```diff
@@ -105,6 +105,7 @@ public void testHybridSearchSequentialClusteringColumns() throws Throwable
     public void testHybridSearchHoleInClusteringColumnOrdering() throws Throwable
     {
         setMaxBruteForceRows(0);
+        QueryController.QUERY_OPT_LEVEL = 0;
```

Comment:
should we reset this to the default value after the test ? 

I see that we are setting QUERY_OPT_LEVEL also in another test

Reply:
Great catch.